LICENSE

```diff
@@ -1,6 +1,7 @@
 ISC License
 
 Copyright (c) 2018-2022, Raedah Group
+Copyright (c) 2023, Cryptopower Developers
```

Comment:
Copyright (c) 2023, The Cryptopower developers

Also, we should start adding the license to each code file starting with this PR. We can do this in batches to ease review. @dreacot what do you think about adding the license to All files that it applies to?

Reply:
Copyright (c) 2023, The Cryptopower developers, this can come before the Raedah Group license since it's more current

Reply:
@ukane-philemon your suggestion is fine, can you provide a template you have in mind?

Reply:
> @ukane-philemon your suggestion is fine, can you provide a template you have in mind?

something simple like we have on dcrdata:
> // Copyright (c) 2023 The Cryptopower developers
   // See LICENSE for details.

Reply:
sounds good, let's do that with files that have changes to them